systemd/org.jfhbrook.plusdeck.conf

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?> <!-- -*- XML -*- -->
+
+<!DOCTYPE busconfig PUBLIC
+ "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
```

Comment:
A challenge here is that other services tend not to have my needs - they tend to create user scoped services, which are then allowed to call into system ones. I don't have that "user" layer, so a lot of the built-in examples won't help me.

Reply:
This isn't true. I was simply misreading the policies.